app/src/main/res/values-fr/strings.xml

```diff
@@ -2,425 +2,302 @@
-
-    <string name="images_to_pdf">Images to PDF</string>
-    <string name="extract_text">Extract Text</string>
-    <string name="pdf_selected">PDF file selected: </string>
-    <string name="snackbar_txtExtracted">Text extracted!</string>
-    <string name="snackbar_pdfselected">PDF file selected!</string>
-    <string name="add_text">Add Text</string>
-    <string name="snackbar_txtselected">Text File selected!</string>
-    <string name="decrypt_protected_file">Selected pdf is password protected. Enter the password to continue...</string>
-    <string name="creating_txt">Creating Text File</string>
-
-    <string name="font_color">Font Color</string>
-    <string name="page_color">Page Color</string>
-    <string name="excel_selected">Excel file selected</string>
-    <string name="excel_to_pdf">Excel to PDF</string>
-    <string name="excel_tv_view_text">Excel file selected: </string>
-    <string name="select_excel_file">Select Excel file</string>
-    <string name="add_to_favourite">Add to Favourite</string>
-    <string name="favourites">Favourites</string>
-
-    <string name="favourites_text">Sélectionnez l\'icône + pour ajouter vos fonctionnalités préférées!</string>
-    <string name="split_one_page_pdf_alert">Le PDF sélectionné ne peut pas être divisé car il ne contient qu\'une page!</string>
+    <string name="zip_to_pdf">ZIP au format PDF</string>
+    <string name="convert_to_pdf">Convertir au format PDF</string>
+    <string name="error_no_image_in_zip">Le fichier ZIP contient pas d\'image</string>
+    <string name="images_to_pdf">Images vers PDF</string>
+    <string name="extract_text">Extraire du texte</string>
+    <string name="pdf_selected">Fichier PDF sélectionné :</string>
+    <string name="snackbar_pdfselected">Fichier PDF sélectionné !</string>
+    <string name="add_text">Ajouter du texte</string>
+    <string name="snackbar_txtselected">Fichier texte sélectionné !</string>
+    <string name="creating_txt">Création du fichier texte</string>
+    <string name="font_color">Couleur de la police</string>
+    <string name="page_color">Couleur de la page</string>
+    <string name="excel_selected">Fichier Excel sélectionné</string>
+    <string name="excel_to_pdf">Excel vers PDF</string>
+    <string name="excel_tv_view_text">Fichier Excel sélectionné :</string>
+    <string name="select_excel_file">Sélectionnez un fichier Excel</string>
+    <string name="add_to_favourite">Ajouter aux favoris</string>
+    <string name="favourites">Favoris</string>
+    <string name="favourites_text">Sélectionnez l\'icône + pour ajouter vos fonctionnalités préférées !</string>
+    <string name="split_one_page_pdf_alert">Le PDF sélectionné ne peut pas être divisé car il ne contient qu\'une seule page !</string>
     <string name="no_excel_file">Aucun fichier Excel sélectionné</string>
-    <string name="split_range_alert">Entered split range will create Selected PDF again!</string>
```

Comment:
You have removed some of the strings @BoFFire 
For example, this one `split_range_alert`?

Reply:
Solved.